### PR TITLE
Fix and test auth config

### PR DESCRIPTION
**Pull request by efirs**

### CODE REVIEW

File: driver/driver.go [hunk 1]
```diff
@@ -280,6 +278,14 @@ func NewDriver(ctx context.Context, cfg *config.Driver) (Driver, error) {
 		cfg.TLS = &tls.Config{MinVersion: tls.VersionTLS12}
 	}
```
Comment (JigarJoshi) [inline driver/driver.go:279]: is the min version to catch this client being used from old go runtime?

**Review by JigarJoshi — APPROVED** [review]

File: driver/driver.go (hunk 1 above)
Comment (efirs) [reply driver/driver.go:279]: This is to prevent connection to insecure servers running TLS1.1 and older

Comment (efirs) [reply driver/driver.go:279]: reported by gosec lint.

Comment (JigarJoshi) [reply driver/driver.go:279]: ok.

**Pull request merged**